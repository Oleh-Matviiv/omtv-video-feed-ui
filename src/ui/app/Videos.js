import React from 'react';
import styled from 'styled-components';

const Main = styled.main.attrs({
  className: 'videos',
})`
  @media screen and (max-width: 1199px) {
    padding: 2rem 0;
  }
  @media screen and (min-width: 1200px) {
    padding: 2rem 4rem;
  }
`;

const Videos = ({ children }) => <Main>{ children }</Main>;

export default Videos;
