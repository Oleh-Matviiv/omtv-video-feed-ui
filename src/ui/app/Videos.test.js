import React from 'react';
import { mount } from 'enzyme';
import 'jest-styled-components';

import Videos, { Main } from './Videos';

describe('snapshots', () => {
  it('renders correctly', () => {
    const wrapper = mount(
      <Videos>
        videos list
      </Videos>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
