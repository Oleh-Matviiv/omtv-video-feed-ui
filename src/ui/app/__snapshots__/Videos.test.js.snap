// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots renders correctly 1`] = `
@media screen and (max-width:1199px) {
  .c0 {
    padding: 2rem 0;
  }
}

@media screen and (min-width:1200px) {
  .c0 {
    padding: 2rem 4rem;
  }
}

<Videos>
  <styled.main>
    <StyledComponent
      forwardedComponent={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [
            Object {
              "className": "videos",
            },
          ],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bdVaJa",
            "isStatic": true,
            "lastClassName": "c0",
            "rules": Array [
              "
  @media screen and (max-width: 1199px) {
    padding: 2rem 0;
  }
  @media screen and (min-width: 1200px) {
    padding: 2rem 4rem;
  }
",
            ],
          },
          "displayName": "styled.main",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-bdVaJa",
          "target": "main",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
    >
      <main
        className="videos c0"
      >
        videos list
      </main>
    </StyledComponent>
  </styled.main>
</Videos>
`;
